code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -666,3 +666,25 @@ trans_to_atmos(var/datum/gas_mixture/target, var/amount=1, var/multiplier=1, var
 /atom/proc/create_reagents(const/max_vol)
 	reagents = new/datum/reagents(max_vol)
 	reagents.my_atom = src
+
+///Used by grinders////
+/datum/reagents/proc/grind_item(var/obj/item/I, var/process = 1)
```

Comment:
Erm? So you mean you think it's harder to maintain if it's scattered in an OOP way?

Honestly I can only see like 3 different overrides, and using `git grep` isn't that hard.